database/index.js

```diff
@@ -1,15 +1,15 @@
 const sampleData = require('./reviewsSampleData.js');
 
 const mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost/Reservations');
+mongoose.connect('mongodb://localhost/reviews');
 
 const db = mongoose.connection;
 db.on('error', console.error.bind(console, 'connection error:'));
 db.once('open', function() {
   // we're connected!
 });
 
-const reviewSchema = mongoose.Schema({
+const listingSchema = mongoose.Schema({
     id: Number,
```

Comment:
make your id unique: true so you don't accidentally add dupes.

Reply:
Done